src/app/app.config.ts

```diff
@@ -4,7 +4,6 @@ import {provideRouter} from '@angular/router';
 import {routes} from './app.routes';
 import {initializeApp, provideFirebaseApp} from "@angular/fire/app";
 import {getAuth, provideAuth} from "@angular/fire/auth";
-import {getAnalytics} from "@angular/fire/analytics";
 
 const firebaseConfig = {
     apiKey: "AIzaSyAtn-g4uiyo9-EFTIPqKgZfOGf3nOSUO_0",
```

Comment:
You need use environments and gitignore, so you aren't exposing your secrets to the public